content/docs/security/security-overview.md

```diff
@@ -46,9 +46,45 @@ Amazon’s secure data centers backed by [AWS Cloud Security](https://aws.amazon
 
 The Microsoft cloud data centers that power Azure focus on high reliability, operational excellence, cost-effectiveness, and a trustworthy online experience for Microsoft customers and partners worldwide. Microsoft regularly tests data center security through both internal and third-party audits. To learn more, refer to [Microsoft's Datacenter security overview](https://learn.microsoft.com/en-us/compliance/assurance/assurance-datacenter-security).
 
-## Compliance
+## Compliance Overview
 
-Neon has successfully completed audits for SOC 2 Type 1 and Type 2, ISO 27001, and ISO 27701, and adheres to GDPR and CCPA regulations. See [Compliance](/docs/security/compliance).
+At Neon, we implement robust technical controls to secure customer and sensitive data in alignment with SOC2, ISO27001, ISO27701 standards and GDPR and CCPA regulations.
+
+All systems are hosted on AWS, where we have implemented specific security measures to protect data. Below is a detailed breakdown of these compliance-relevant security measures for access control, encryption, network security, event logging, vulnerability management, backups, data deletion and retention:
```

Comment:
We now have projects on Azure. Does this refer to internal systems only or should we adjust the content to address each item from an Azure perspective?

Reply:
I'll include Azure after the SOC2 audit in November. Let's keep it out for now to prevent confusion. 

https://neondb.slack.com/archives/C03GTPNTTHQ/p1728655452410279?thread_ts=1728637738.939789&cid=C03GTPNTTHQ